src/components/views/rooms/EventTile.tsx

```diff
@@ -918,6 +918,7 @@ export class UnwrappedEventTile extends React.Component<EventTileProps, IState>
     public render(): ReactNode {
         const msgtype = this.props.mxEvent.getContent().msgtype;
         const eventType = this.props.mxEvent.getType();
+        const id = `mx_eventTile_${this.props.mxEvent.getId()!}`;
```

Comment:
An event may be rendered multiple times one on screen breaking the ID uniqueness constraint in the DOM, e.g. pinned events, thread roots in both main timeline and thread view